html/elements/tr.json

```
@@ -194,7 +194,7 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
+                "version_added": "≤72",
```

We use 1 for other `valign` data as well (`td` for example)

```suggestion
                "version_added": "1",
```